hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -37,6 +37,8 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 
+import org.apache.hadoop.fs.RemoteIterator;
```

Comment:
needs to go into the non-shaded bit of hadoop imports.

Reply:
Done